prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -2198,6 +2201,23 @@ subroutine initProgramVariables(input, env)
       call init(pMDIntegrator, pVelocityVerlet)
     end if
 
+    ! Initialising plumed
+    tPlumed = input%ctrl%tPlumed
```

Comment:
Should probably test here if this is an internally driven MD if (tPlumed), and not some other type of calculation

Reply:
What do you exactly mean by that?

Reply:
If the parser is not used to initialise the input structure(Materials studio), could get here with input%ctrl%tPlumed being true for something that is not an MD run.